src/Security/Security.php

```diff
@@ -1107,7 +1107,7 @@ public static function clear_default_admin()
      * @param string $password The password (in cleartext)
      * @return bool True if successfully set
      *
-     * @deprecated 4.0.0:5.0.0 Please use DefaultAdminService::setDefaultAdmin($username, $password)
+     * @deprecated 4.0.0 Please use DefaultAdminService::setDefaultAdmin($username, $password)
```

Comment:
Update the notice message and version as well.
```suggestion
     * @deprecated 4.0.0 Use DefaultAdminService::setDefaultAdmin($username, $password) instead
```